### PR TITLE
hardware-enablement.rst: fix wording

### PR DESCRIPTION
**Pull request by ewentual**

### CODE REVIEW

File: supplementary/hardware-enablement.rst [hunk 1]
```diff
@@ -45,7 +45,7 @@ For all ARM based smart phones and tablets this should be inside
 
    root/arch/arm/boot/dts/
 
-or if your using 64-bit arm64 (e.g. BQ Aquaris M10) all files are sorted by manufacturer inside
+or if your device is using a 64-bit (arm64) processor (e.g. BQ Aquaris M10) all files are sorted by manufacturer inside
```
Comment (vanyasem) [inline supplementary/hardware-enablement.rst]: I would say "is running on", the wording is quite unusual for me

Comment (vanyasem) [reply supplementary/hardware-enablement.rst]: I think the original author wanted to say "if you're using a 64-bit processor", which is why it got so weird now

Comment (vanyasem) [reply supplementary/hardware-enablement.rst]: Nvm, an English-man said that it sounds fine

Comment (abhishek9650) [reply supplementary/hardware-enablement.rst]: "or if your device is uses an arm64 processor (e.g. BQ Aquaris M10) all the files are sorted by the manufacturer in :"
this is what i think it should be

**Pull request merged**